### PR TITLE
Submission upload

### PR DESCRIPTION
**Pull request by virasakd**
The SubmissionsUpload branch contains an updated media upload component, video hosting capabilities, a video verification system within the submission page, a ReactPlayer Component, and Iframe APIs.

### CODE REVIEW

File: site/gatsby-site/gatsby-config.js
Comment (ShaurGaur) [inline site/gatsby-site/gatsby-config.js:1]: Shouldn't we remove the hard-coded Cloudinary info here, and go back to the defaults? @INeedNZT @virasakd

File: site/gatsby-site/src/components/forms/PreviewImageInputGroup.js
Comment (ShaurGaur) [inline site/gatsby-site/src/components/forms/PreviewImageInputGroup.js:1]: Remove comments here?

File: site/gatsby-site/src/components/forms/SubmitForm.js
Comment (ShaurGaur) [inline site/gatsby-site/src/components/forms/SubmitForm.js:1]: I added some console.logs when trying to debug the form stuff, but forgot to remove them sorry. Do you want to remove them real quick? If not, I can.

File: site/gatsby-site/src/utils/cloudinary.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
+// import React, { useState, useRef, useEffect } from 'react';
+// import { AdvancedImage, lazyload } from '@cloudinary/react';
+// import { CloudinaryImage } from '@cloudinary/base';
+// import { format, quality } from '@cloudinary/base/actions/delivery';
+// import { auto } from '@cloudinary/base/qualifiers/format';
+// import { auto as qAuto } from '@cloudinary/base/qualifiers/quality';
+// import config from '../../config';
+// import PlaceholderImage from 'components/PlaceholderImage';
+
```
Comment (ShaurGaur) [inline site/gatsby-site/src/utils/cloudinary.js:9]: Let's remove the comments here too.

**Review by ShaurGaur — COMMENTED** [review]
Looks pretty good so far. I left a few comments, mostly about console logs and comments we left in, but also for the hard-coded Cloudinary database. 

Also there's some merge conflicts in some files. What's up with those? I can't tell from GitHub.

File: site/gatsby-site/src/components/forms/SubmitForm.js
Comment (virasakd) [reply site/gatsby-site/src/components/forms/SubmitForm.js:1]: Comments and print statements have been removed in the last commit.

File: site/gatsby-site/src/utils/cloudinary.js (hunk 1 above)
Comment (virasakd) [reply site/gatsby-site/src/utils/cloudinary.js]: Comments have been removed in the last commit.

File: site/gatsby-site/src/components/forms/PreviewImageInputGroup.js
Comment (virasakd) [reply site/gatsby-site/src/components/forms/PreviewImageInputGroup.js:1]: Comments have been removed in the last commit.

File: site/gatsby-site/gatsby-config.js
Comment (virasakd) [reply site/gatsby-site/gatsby-config.js:1]: Yes, I removed my account.

**Review by ShaurGaur — CHANGES_REQUESTED** [review]
Looks pretty good. Sounds like we just need to fix the merge conflicts.

Also let's do a new PR with staging instead of master.

**Comment by ShaurGaur** [comment]
Closed because branch is out of date and we're merging with staging instead.

**Pull request closed without merging**